src/infection-model/seir.tsx

```diff
@@ -225,6 +216,27 @@ function getCurveProjections(inputs: SimulationInputs & CurveProjectionInputs) {
     rateOfSpreadFactor,
   } = inputs;
 
+  // 3d array. D1 = SEIR compartment. D2 = day. D3 = age bracket
+  const projectionGrid = ndarray(
+    new Array(seirIndexList.length * numDays + 1 * ageGroupIndex.__length).fill(
```

Comment:
yeah good catch ... luckily it didn't really affect anything though, because all these values are supposed to get overwritten anyway. this is just a backstop to prevent any `undefined`s from sneaking into the arithmetic